dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp

```diff
@@ -473,11 +473,11 @@ var editButtonRow="editContentletButtonRow";
 		 		request.setAttribute(com.dotmarketing.util.WebKeys.PERMISSIONABLE_EDIT, fatty);
 		}%>
 
-		<div id="versions" dojoType="dijit.layout.ContentPane" title="<%= LanguageUtil.get(pageContext, "History") %>" onShow="refreshVersionCp();">
-			<div id="contentletVersionsDiv" style="height:100%;">
+		<div id="versions" class="history" dojoType="dijit.layout.ContentPane" title="<%= LanguageUtil.get(pageContext, "History") %>" onShow="refreshVersionCp();">
+			<div id="contentletVersionsDiv" style="height:100%;" class="content-edit__history-version">
 			</div>
-
-			<div>
+			<hr class="history__divider">
+			<div class="content-edit__history-status">
```

Comment:
If you are "inside" the ```history``` block, this classname should be ```history__status``` I think ```history``` belongs to anothers places beside ```content-edit```